ai_feedback/helpers/template_utils.py

```diff
@@ -2,9 +2,9 @@
 import re
 import sys
 from pathlib import Path
-from string import Template
-from typing import List, Optional
+from typing import Any, Dict, List, Optional, Tuple
 
+import fitz
```

Comment:
Seems like this is an old name (https://pymupdf.readthedocs.io/en/latest/tutorial.html). The package should also be added to the project dependencies